tasks/pipeline.py

```diff
@@ -810,6 +810,27 @@ def update_circleci_config(file_path, image_tag, test_version):
         circle.write(circle_ci.replace(f"{match.group(0)}", f"{image}:{image_tag}\n"))
 
 
+@task(
+    help={
+        "file_path": "path of the CircleCI configuration YAML file",
+    },
+    autoprint=True,
+)
+def get_gitlab_config_image_tag(_, file_path=".gitlab-ci.yml"):
+    """
+    Print the current image tag of the given CircleCI configuration file (default: ".circleci/config.yml")
```

Comment:
Thanks for addressing my comment!
You probably need to update the comment and the help of the function to reflect the change